updater/sponsorships.py

```diff
@@ -54,44 +58,49 @@ def update():
             continue
         for sponsored in sponsoreds:
             context_id = sponsored['args']['addr'].lower()
-
             print('checking sponsored\tapp_name: {0}, context_id: {1}'.format(
                 app['_key'], context_id))
-            context = contexts[app['context']]
-            collection = context['collection']
-            c = db[collection].find(
-                {'contextId': context_id})
-            if c.empty():
-                print("the context id doesn't link to any user under this context")
-                continue
-            user = c.next()['user']
 
             # remove testblocks if exists
             testblocks.delete_match({
-              'contextId': context_id,
-              'action': 'sponsorship',
-              'app': app['_key']
+                'contextId': context_id,
+                'action': 'sponsorship',
+                'app': app['_key']
             })
 
-            c = sponsorships.find(
```

Comment:
This makes sense we keep this check and not waste gas to send `Sponsor` operation for already sponsored user.

Reply:
It's done